composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "ampconf/theme",
+    "description": "WordPress AMPConf theme",
```

Comment:
Maybe this should be `"WordPress AMP theme"`

Reply:
Should there also be a `composer.lock` file, like [in the AMPConf theme](https://github.com/xwp/ampconf/pull/4/files#diff-1da2c7edc898c70e5a79a9997c98ceccR1)?

Reply:
Thanks for the heads up, this is fixed in this commit d05ba50ac2eb474904b79a950da8ff6be45dc01b